app/uk/gov/hmrc/customsservicestatus/repositories/AdminCustomsServiceStatusRepository.scala

```diff
@@ -16,23 +16,28 @@
 
 package uk.gov.hmrc.customsservicestatus.repositories
 
+import cats.data.OptionT
 import com.mongodb.client.model.Indexes.ascending
 import org.mongodb.scala.*
 import org.mongodb.scala.bson.BsonDateTime
 import org.mongodb.scala.model.*
-import org.mongodb.scala.model.Filters.gte
+import org.mongodb.scala.model.Filters.{equal, gte}
 import uk.gov.hmrc.customsservicestatus.models.OutageData
+import org.mongodb.scala.result.InsertOneResult
 import uk.gov.hmrc.mongo.MongoComponent
+import uk.gov.hmrc.mongo.play.json.Codecs.JsonOps
 import uk.gov.hmrc.mongo.play.json.PlayMongoRepository
 import uk.gov.hmrc.play.http.logging.Mdc
 
 import java.time.Instant
+import java.util.UUID
 import javax.inject.{Inject, Singleton}
 import scala.concurrent.{ExecutionContext, Future}
 
 @Singleton
 class AdminCustomsServiceStatusRepository @Inject() (
-  mongo: MongoComponent
+  mongo:                     MongoComponent,
+  archivedOutagesRepository: ArchivedOutagesRepository
```

Comment:
Discussed offline and agreed that the `ArchivedOutagesRepository` should be moved up a level to the service class above, which would then have both repositories as delegates. This would allow the `AdminCustomsServiceStatusRepository` to be responsible solely for interacting with its underlying collection.

The `archive` function within the service would then delete the record from the normal outages table and add it to the archive table (by calling each of its repository delegates in turn).